packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -374,15 +377,17 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
         tabIndex={ 0 }
         onKeyDown={ this.props.disabled ? undefined : this._onSplitButtonKeyDown }
       >
+
         <span aria-hidden={ true } style={ { 'display': 'flex' } }>
           { this._onRenderContent(tag, buttonProps) }
-          { this._onRenderSplitButtonMenuButton() }
+          { classNames && classNames.divider && <span className={ classNames.divider } /> }
```

Comment:
Nit: for clarity and consistency, maybe it would be better to use a function here: `_onRenderSplitButtonDivider`.